cs-amazon/src/main/java/io/cloudslang/content/amazon/factory/helpers/NetworkUtils.java

```diff
@@ -93,6 +96,17 @@
         return queryParamsMap;
     }
 
+    public Map<String, String> getCreateSubnetQueryParamsMap(InputsWrapper wrapper) {
+        Map<String, String> queryParamsMap = new LinkedHashMap<>();
```

Comment:
@Bonczidai yes... is true... I preferred to use LinkedHashMap when I had to compose a much more complicated query... here is not the case. I will change it.